ansible/vars/dev-vagrant.user.yml

```diff
@@ -8,4 +8,4 @@
 # Require confirmation of the email address of newly registered users.
 # Requires a working mailer setup. Add SMTP credentials in
 # secrets/dev-vagrant.secrets.yml before enabling this setting.
-#site_user_email_confirmation_enabled: true
+#site_user_email_confirmation_enabled: true
```

Comment:
Configure your IDE to respect eol settings from `.editorconfig`. There must be a plugin available